src/elements/content-sidebar/versions/__tests__/VersionsSidebar.test.js

```diff
@@ -28,6 +28,13 @@ describe('elements/content-sidebar/versions/VersionsSidebar', () => {
             expect(wrapper).toMatchSnapshot();
         });
 
+        test('should show an upsell inline error if the errorUpsell prop is provided', () => {
+            const wrapper = getWrapper({ error: messages.versionFetchErrorUpsell, errorUpsell: true, versions: [] });
+
+            expect(wrapper.exists(InlineError)).toBe(true);
+            expect(wrapper).toMatchSnapshot();
```

Comment:
We've been trying to reduce our dependence on snapshot testing. Can we make explicit assertions, instead?